pkg/codegen/python/python.go

```diff
@@ -119,8 +119,159 @@ func PyName(name string) string {
 				continue
 			}
 
-			// We want to fold digits immediately following an acronym into the same
-			// component as the acronym.
+			// We want to fold digits or the lowercase letter 's' immediately following
+			// an acronym into the same component as the acronym.
+			if unicode.IsDigit(char) || char == 's' {
```

Comment:
Is it possible to share more of the code? Maybe something like:

```go
func PyName(name string) string {
	return pyNameImpl(name, false /*legacy*/)
}

func PyNameLegacy(name string) string {
	return pyNameImpl(name, true /*legacy*/)
}

// Better name ideas?
func pyNameImpl(name string, legacy bool) string {
	...
}
```

Reply:
Hah yeah that's a much better idea!